### PR TITLE
MF-744: Tablet mode-Expand Date and time column to fit - not wrapped

### PR DESCRIPTION
**Pull request by KorirC**
## Requirements

- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d5df24b2bf4bbb58b866b0).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

- Tablet mode:Expand Date and time column to fit - not wrapped

## Screenshots
vitals3
vitals4

## Related Issue

Issue: [MF-744](https://issues.openmrs.org/browse/MF-744)

## Other

*None.*

### CODE REVIEW

**Review by FlorianRappl — APPROVED** [review]

**Comment by HerbertYiga** [comment]
thanks @KorirC  since this is approved you can remove it from the draft

**Comment by KorirC** [comment]
> thanks @KorirC since this is approved you can remove it from the draft

I realized the above implementation affects the size of vitals workspace, working on a fix for that and still have the date and time column no wrapped.

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss [hunk 1]
```diff
@@ -97,3 +97,13 @@
     border-left: 0rem;
   }
 }
+
+.customTable  tbody  tr:nth-child(even)  td {
+  background-color: #f4f4f4;
```
Comment (FlorianRappl) [inline packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss:102]: What's f4f4f4?

File: packages/esm-patient-vitals-app/src/vitals/vitals-overview.scss [hunk 2]
```diff
@@ -106,3 +106,12 @@
   justify-content: end;
 }
 
+.customTable  tbody  tr:nth-child(even)  td {
+  background-color: #f4f4f4;
```
Comment (FlorianRappl) [inline packages/esm-patient-vitals-app/src/vitals/vitals-overview.scss:110]: Same here - why repeat? Can we make this reusable?

Comment (KorirC) [reply packages/esm-patient-vitals-app/src/vitals/vitals-overview.scss:110]: Yes

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss [hunk 3]
```diff
@@ -97,3 +97,13 @@
...
+}
```
Comment (brandones) [inline packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss:103]:
```suggestion
```

File: packages/esm-patient-vitals-app/src/vitals/vitals-overview.scss [hunk 4]
```diff
@@ -106,3 +106,12 @@
...
+}
+
```
Comment (brandones) [inline packages/esm-patient-vitals-app/src/vitals/vitals-overview.scss:112]:
```suggestion
```

**Review by brandones — APPROVED** [review]

**Pull request merged**